chainindex/ddls.go

```diff
@@ -22,7 +22,7 @@ const (
 
 	stmtUpdateEventsToRevertedFromHeight = "UPDATE event SET reverted = 1 WHERE message_id IN (SELECT message_id FROM tipset_message WHERE height >= ?)"
 
-	stmtIsTipsetMessageNonEmpty = "SELECT EXISTS(SELECT 1 FROM tipset_message LIMIT 1)"
+	stmtIsIndexEmpty = "SELECT NOT EXISTS(SELECT 1 FROM tipset_message LIMIT 1)"
```

Comment:
I think we should add `LIMIT 1` to other `EXIST` queries for optimisation.

Reply:
@akaladarshi Please feel free to take over this PR and make whatever changes you need.